Testing/SystemTests/tests/framework/ISIS_WISHSingleCrystalReduction.py

```diff
@@ -132,8 +132,147 @@ def validate(self):
+        van = load_data_and_normalise('WISH/input/11_4/WISH00019612.raw', outputWorkspace="van")
+        # create Abs Correction for V
+        shape = '''<sphere id="V-sphere">
+            <centre x="0.0"  y="0.0" z="0.0" />
+            <radius val="0.0025"/>
+            </sphere>'''
+        CreateSampleShape(InputWorkspace=van, ShapeXML=shape)
+        SetSampleMaterial(InputWorkspace=van, SampleNumberDensity=0.0119, ScatteringXSection=5.197,
+                          AttenuationXSection=4.739, ChemicalFormula='V0.95 Nb0.05')
+        abs_cor = AbsorptionCorrection(InputWorkspace=van, ElementSize=0.5)
+        # correct Vanadium run for absorption
+        van = Divide(LHSWorkspace=van, RHSWorkspace=abs_cor, OutputWorkspace=van)
+        # smooth data
+        SmoothNeighbours(InputWorkspace=van, OutputWorkspace=van, Radius=3)
+        SmoothData(InputWorkspace=van, OutputWorkspace=van, NPoints=300)
+
+    def validate(self):
+        return "van", "WISH19612_vana_bank1_SXProcessed.nxs"
+
+
+class WISHNormaliseDataAndCreateMDWorkspaceTest(MantidSystemTest):
+    """
+    This tests the loading and normalisation of data, incl. correction for detector efficiency using vanadium and
+    creation of MD workspace
+    """
+
+    def cleanup(self):
+        ADS.clear()
+
+    def runTest(self):
+        # Load processed vanadium for normalisation (bank 1)
+        van = LoadNexus(Filename="WISH19612_vana_bank1_SXProcessed.nxs")
+        # Load raw data (bank 1)
+        ws = load_data_and_normalise("WISH00038237.raw")  # default so doesn't get overwrite van
+        # normalise to vanadium
+        RebinToWorkspace(WorkspaceToRebin=van, WorkspaceToMatch=ws, OutputWorkspace=van)
+        Divide(LHSWorkspace=ws, RHSWorkspace=van, OutputWorkspace=ws)
+        ReplaceSpecialValues(InputWorkspace=ws, OutputWorkspace=ws, NaNValue=0,
+                             InfinityValue=0, BigNumberThreshold=1e15, SmallNumberThreshold=-1e15)
+        # Convert to Diffraction MD and Lorentz Correction
+        wsMD = ConvertToDiffractionMDWorkspace(InputWorkspace=ws, LorentzCorrection=True, OneEventPerBin=False)
+        # BinMD to 2D object and convert to histo so can compare saved workspace
+        wsMD_2Dcut = BinMD(InputWorkspace=wsMD, AxisAligned=False, BasisVector0='Q_lab_x,Angstrom^-1,1.0,0.0,0.0',
+                           BasisVector1='Q_lab_y,Angstrom^-1,0.0,1.0,0.0',
+                           BasisVector2='Q_lab_z,Angstrom^-1,0.0,0.0,1.0',
+                           OutputExtents='0.2,0.8,-0.4,0.4,0.05,0.1', OutputBins='50,50,1')
+        ConvertMDHistoToMatrixWorkspace(InputWorkspace=wsMD_2Dcut, outputWorkspace="wsHisto_2Dcut")
+
+    def validate(self):
+        return "wsHisto_2Dcut", "WISH38237_MD_2Dcut.nxs"
+
+
+class WISHIntegrateSatellitePeaksTest(MantidSystemTest):
+    """
+    This tests the loading and normalisation of data, incl. correction for detector efficiency using vanadium and
+    creation of MD workspace
```

Comment:
Is this comment applicable? Wasn't sure if it was a copy and paste from the previous test?

Reply:
Yes sorry, fixed that now